linux_os/guide/system/auditing/policy_rules/audit_delete_failed/rule.yml

```diff
@@ -56,3 +56,15 @@ template:
         filepath: /etc/audit/rules.d/30-ospp-v42-4-delete-failed.rules
         contents: |+
             {{{ file_contents_audit_delete_failed|indent(12) }}}
+
```

Comment:
this file is not relevant anymore for this SRG, it was updated recently... please rebase on top of master

Reply:
it was removed here: https://github.com/ComplianceAsCode/content/pull/8642